devito/ir/ietxdsl/cluster_to_ssa.py

```diff
@@ -275,9 +278,12 @@ def _build_step_loop(
             raise ValueError("step must be int!")
 
         iter_args = list(self.function_args.values())
+
         # Create the for loop
-        loop = scf.For(lb, arith.Addi(ub, one), step, iter_args, Block(arg_types=[builtin.IndexType(), *(a.type for a in iter_args)]))
-        loop.body.block.args[0].name_hint = "time"
+        loop = scf.For(lb, arith.Addi(ub, step), step, iter_args, Block(arg_types=[builtin.IndexType(), *(a.type for a in iter_args)]))
```

Comment:
Are we sure step is always 1?
If so, can we rename the reused value to `one`? I'd find it more readable
We can write step=one in the constructor

Reply:
It may not always be one